led_strip/include/led_strip_types.h

```diff
@@ -11,6 +11,8 @@
 extern "C" {
 #endif
 
+enum led_type_t { LED_TYPE_WS2812 = 0, LED_TYPE_SK6812 };
```

Comment:
@suda-morris, it sounds pretty sensible.
Indeed, the bunch of datasheets for these things is a real mes<DEL><DEL><DEL> complex matter.
I analyzed a few dozens here: https://docs.google.com/spreadsheets/d/1-AGjwhz6Os8QdFERwoYQDjFJy2YC0yVbzDxkl0ICr7E/edit?usp=sharing
There's not an easy way out, better stick on the simple concept, like the pixel format.
If you don't mind, I'd rather prefer using LED_PIXEL_FORMAT_GRB, LED_PIXEL_FORMAT_GRBW, since I found some datasheets showing an R, G, B byte sequence.
About the bit timing management, I'll leave the two configurations, but after the datasheets analysis I think it would be better to provide a single default sound timing (or two, a fast one and a more conservative one) and give the user the opportunity to customize it in case of problems.